config/locales/idv/en.yml

```diff
@@ -35,7 +35,6 @@ en:
       use_financial_account: Enter one of these here >
       zipcode: ZIP Code
     index:
-      cancel_link: No, I don’t have it
```

Comment:
Should we allow optionally overriding the text of the cancel link? Or do we we always want it to be plain "cancel" ?